### DISCUSSION TITLE
TimePicker: steps of 30 minutes

### DISCUSSION THREAD

**Discussion by goyne** [filamentphp/filament#12179] [Help]
### Package

Form builder

### Package Version

v3.2.62

### How can we help you?

I'm trying to configure my TimePicker component to show steps of 30 minutes. For some reason the component ignores my config settings...

`Components\TimePicker::make('starts_at')->columnSpan(6)->label(__('Time'))->minutesStep(30)->seconds(false)`

It's a fresh Laravel environment (11.2.0) with the latest Filament version.

Am I overlooking something? Thanks.

![Scherm­afbeelding 2024-04-04 om 09 48 44](https://github.com/filamentphp/filament/assets/2893526/4df9385a-6dbc-4bda-a6cc-33768f56f202)

**Comment by goyne** [comment]
No one else who's run into this?

**Reply by Knixie** [reply]
I'd run into this, do you have any solution now?

**Reply by Filippo-Bonesso** [reply]
@Knixie @goyne Seems only working when using ->native(false) on timepicker, not well specified in the documentation btw
